### DISCUSSION TITLE
Buff les items de vie

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#915] [Idées acceptées / déjà implémentées]
(pas les popos, les da)
Parce que là ils sont vraiment éclatés

**Comment by DoctorT1807** [comment]
Je trouve pas vraiment, après sa dépend surtout de tes pv en tant que cannonier le da +6pv je le trouvait très bien après avec les mini-event ils sont moins utile c sûr

**Reply by Pietagorh** [reply]
Bah franchement si tu compares à ceux d'argent c'est ridicule
Le meilleur légendaire de vie donne 10 de vie chaque jour
Le meilleur légendaire d'argent donne 466:moneybag: chaque jour, soit un peu plus que le prix d'une Bizous'koup, qui donne 70 de vie

**Reply by Pietagorh** [reply]
Sachant qu'en plus, l'argent sert à d'autres choses

**Reply by BastLast** [reply]
en lisant ca j'en déduit que les items d'argent sont pétés

**Reply by Pietagorh** [reply]
Non, vu que tout coûte cher x)

**Reply by QQtin3** [reply]
Les items d'argent sont loin d'être broken, déjà il faut l'avoir et de plus 466  pour un légendaire c'est ok je trouve, c'est pas tout les jours qu'on en a un ^^

**Reply by BastLast** [reply]
certe mais ca serait dommage qu'un joueur ai un bonus trop important 
![image](https://user-images.githubusercontent.com/38099970/154810600-5d348ace-029d-444d-b798-83cf011c6e7b.png)
Voilà la progression des gains de money

**Reply by BastLast** [reply]
je pense que les items légendaires donnent actuellement un bonus un peu trop important

**Reply by BastLast** [reply]
![image](https://user-images.githubusercontent.com/38099970/154810703-1abcb279-b9e3-4f6e-bb90-0078624c4b9c.png)
en réduisant un petit peu déjà, c'est plus logique comme progression

**Reply by Pietagorh** [reply]
Mais de toute façon, tout le monde le dira, les items de vie sont pas assez forts
